cli/azd/pkg/tools/sqlcmd/sqlcmd.go

```diff
@@ -0,0 +1,376 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package sqlcmd
+
+import (
+	"archive/tar"
+	"archive/zip"
+	"compress/bzip2"
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"log"
+	"net/http"
+	"os"
+	"path/filepath"
+	"runtime"
+	"strings"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/policy"
+	"github.com/azure/azure-dev/cli/azd/internal/tracing"
+	"github.com/azure/azure-dev/cli/azd/internal/tracing/events"
+	"github.com/azure/azure-dev/cli/azd/pkg/config"
+	"github.com/azure/azure-dev/cli/azd/pkg/exec"
+	"github.com/azure/azure-dev/cli/azd/pkg/input"
+	"github.com/azure/azure-dev/cli/azd/pkg/osutil"
+	"github.com/azure/azure-dev/cli/azd/pkg/tools"
+	"github.com/blang/semver/v4"
+)
+
+// sqlCmdCliVersion is the minimum version of sqlCmd cli that we require
+var sqlCmdCliVersion semver.Version = semver.MustParse("1.8.0")
+
+func NewSqlCmdCli(ctx context.Context, console input.Console, commandRunner exec.CommandRunner) (*SqlCmdCli, error) {
```

Comment:
NIT: Let's name the exported CLI type, `Cli` and this method `NewCli` as we are now doing for the other tools.